plugins/item_tasks/web_client/models/WidgetModel.js

```diff
@@ -42,6 +42,9 @@ import tinycolor from 'tinycolor2';
  *   * new-file:
  *      an output file (contains an existing folder id and a
  *      name that will be used for the new item.
+ *   * new-directory:
```

Comment:
Do we really want to call this "directory"? Everything else in Girder uses the terminology of "folders".